gateway/api/views.py

```diff
@@ -187,6 +190,67 @@ def stop(self, request, pk=None):  # pylint: disable=invalid-name,unused-argumen
         return Response({"message": message})
 
 
+class FilesViewSet(viewsets.ViewSet):
+    """ViewSet for file operations handling.
+
+    Note: only tar files are available for list and download
+    """
+
+    BASE_NAME = "files"
+
+    # @action(methods=["GET"], detail=False)
```

Comment:
```suggestion
    @action(methods=["GET"], detail=False)
```
?

Reply:
Forgot to remove it. `list` is a reserved django action, therefore it does not need decorator